openbas-api/src/main/java/io/openbas/rest/atomic_testing/form/InjectResultOverviewOutput.java

```diff
@@ -42,52 +37,49 @@ public class InjectResultDTO {
   @JsonProperty("inject_commands_lines")
   private InjectStatusCommandLine commandsLines;
 
-  @JsonProperty("inject_expectations")
-  private List<InjectExpectation> expectations;
-
   @JsonProperty("inject_type")
   private String type;
 
-  @Schema(description = "Kill Chain Phases")
-  @JsonProperty("inject_kill_chain_phases")
-  @NotNull
-  private List<KillChainPhase> killChainPhases;
+  @JsonProperty("injects_tags")
+  private List<String> tagIds;
 
-  @Schema(description = "Attack Patterns")
-  @JsonProperty("inject_attack_patterns")
-  @NotNull
-  private List<AttackPattern> attackPatterns;
+  @JsonProperty("injects_documents")
+  private List<String> documentIds;
 
   @Schema(description = "Full contract")
   @JsonProperty("inject_injector_contract")
   @NotNull
-  private InjectorContract injectorContract;
+  private InjectorContractSimple injectorContract;
 
   @JsonProperty("inject_status")
-  private InjectStatus status;
+  private InjectStatusOutput status;
 
-  @Schema(
-      description = "Specifies the categories of targetResults for atomic testing.",
-      example = "assets, asset groups, teams, players")
-  @JsonProperty("inject_targets")
-  @NotNull
-  private List<InjectTargetWithResult> targets;
-
-  @Default
-  @Schema(description = "Result of expectations")
-  @JsonProperty("inject_expectation_results")
-  @NotNull
-  private List<ExpectationResultsByType> expectationResultByTypes = new ArrayList<>();
+  @JsonProperty("inject_expectations")
+  private List<InjectExpectationSimple> expectations;
 
-  @JsonProperty("injects_tags")
-  private List<String> tagIds;
+  @JsonProperty("inject_kill_chain_phases")
+  private List<KillChainPhaseSimple> killChainPhases;
 
-  @JsonProperty("injects_documents")
-  private List<String> documents;
+  @JsonProperty("inject_attack_patterns")
+  private List<AttackPatternSimple> attackPatterns;
 
   @JsonProperty("inject_ready")
   private Boolean isReady;
 
   @JsonProperty("inject_updated_at")
-  private Instant updatedAt = now();
+  private Instant updatedAt;
+
+  // -- COMPUTED ATTRIBUTES --
+
+  @Default
+  @Schema(description = "Result of expectations")
```

Comment:
**nitpick :** We should have @Schema on all the properties, not only the computed ones.